tests/system/providers/google/cloud/compute/example_compute.py

```diff
@@ -0,0 +1,274 @@
+    gce_instance_start = ComputeEngineStartInstanceOperator(
+        task_id='gcp_compute_start_task',
+        project_id=PROJECT_ID,
+        zone=LOCATION,
+        resource_id=GCE_INSTANCE_NAME,
+    )
+    # [END howto_operator_gce_start]
+
+    # Duplicate start for idempotence testing
+    # [START howto_operator_gce_start_no_project_id]
+    gce_instance_start2 = ComputeEngineStartInstanceOperator(
+        task_id='gcp_compute_start_task2',
+        zone=LOCATION,
+        resource_id=GCE_INSTANCE_NAME,
+    )
+    # [END howto_operator_gce_start_no_project_id]
+
+    # [START howto_operator_gce_stop]
+    gce_instance_stop = ComputeEngineStopInstanceOperator(
+        task_id='gcp_compute_stop_task',
+        project_id=PROJECT_ID,
+        zone=LOCATION,
+        resource_id=GCE_INSTANCE_NAME,
+    )
+    # [END howto_operator_gce_stop]
+    gce_instance_stop.trigger_rule = TriggerRule.ALL_DONE
+
+    # Duplicate stop for idempotence testing
+    # [START howto_operator_gce_stop_no_project_id]
+    gce_instance_stop2 = ComputeEngineStopInstanceOperator(
+        task_id='gcp_compute_stop_task2',
+        zone=LOCATION,
+        resource_id=GCE_INSTANCE_NAME,
+    )
+    # [END howto_operator_gce_stop_no_project_id]
+    gce_instance_stop2.trigger_rule = TriggerRule.ALL_DONE
+
+    # [START howto_operator_gce_set_machine_type]
+    gce_set_machine_type = ComputeEngineSetMachineTypeOperator(
+        task_id='gcp_compute_set_machine_type',
+        project_id=PROJECT_ID,
+        zone=LOCATION,
+        resource_id=GCE_INSTANCE_NAME,
+        body={'machineType': f'zones/{LOCATION}/machineTypes/{SHORT_MACHINE_TYPE_NAME}'},
+    )
+    # [END howto_operator_gce_set_machine_type]
+
+    # Duplicate set machine type for idempotence testing
+    # [START howto_operator_gce_set_machine_type_no_project_id]
+    gce_set_machine_type2 = ComputeEngineSetMachineTypeOperator(
+        zone=LOCATION,
+        resource_id=GCE_INSTANCE_NAME,
+        body={'machineType': f'zones/{LOCATION}/machineTypes/{SHORT_MACHINE_TYPE_NAME}'},
+        task_id='gcp_compute_set_machine_type2',
+    )
+    # [END howto_operator_gce_set_machine_type_no_project_id]
+
+    # [START howto_operator_gce_delete_no_project_id]
+    gce_instance_delete = ComputeEngineDeleteInstanceOperator(
+        task_id='gcp_compute_delete_instance_task',
+        zone=LOCATION,
+        resource_id=GCE_INSTANCE_NAME,
+    )
+    # [END howto_operator_gce_delete_no_project_id]
+    gce_instance_delete.trigger_rule = TriggerRule.ALL_DONE
+
+    # [START howto_operator_gce_delete_no_project_id]
+    gce_instance_delete2 = ComputeEngineDeleteInstanceOperator(
+        task_id='gcp_compute_delete_instance_task_2',
+        zone=LOCATION,
+        resource_id=GCE_INSTANCE_NAME,
+    )
+    # [END howto_operator_gce_delete_no_project_id]
+    gce_instance_delete.trigger_rule = TriggerRule.ALL_DONE
+
+    bash_wait_operator = BashOperator(task_id="delay_bash_task", bash_command="sleep 3m")
```

Comment:
Why do we need 3 min waits, is the operator non blocking?

Reply:
I have added this logic, because Operators for creating instances and deleting them need time to finish their execution, but instances are not ready yet, as it takes time to finish this process on gcloud side. So it leads to errors "instance not found" on next steps while changing that newly created instances